core/chains/evm/logpoller/log_poller_test.go

```diff
@@ -492,3 +493,70 @@ func TestLogPoller_MergeFilter(t *testing.T) {
 	_, err := lp.Logs(1, 1, EmitterABI.Events["Log1"].ID, common.HexToAddress("0x2ab9a2dc53736b361b72d900cdf9f78f9406fbbd"))
 	require.Error(t, err)
 }
+
+func TestLogPoller_GetBlocks(t *testing.T) {
```

Comment:
could include a ctx cancellation scenario as well

Reply:
ah it seems like `SimulatedBackend` doesn't honor context cancellations/timeouts if that's what you meant.

Reply:
added test cases